src/ModuleServiceProvider.php

```diff
@@ -259,6 +264,9 @@ protected function determineisActive() : void
 			if (static::class === $moduleIndicator || ($targetModule && $this->getModuleFromString(static::class) === $targetModule)) {
 				$this->isActive     = true;
 				self::$activeModule = $targetModule;
+			} elseif (!$targetModule) {
+				$this->isActive     = false;
+				self::$activeModule = 'nomodule';
```

Comment:
Should be null i stead of a predefined string or the string should be defined as a constant

Reply:
updated. null would indicate that no activeModule is found yet so a const is used instead.